Tests/test_util.py

```diff
@@ -35,6 +35,18 @@ def test_is_path(self):
         # Assert
         self.assertTrue(it_is)
 
+    @unittest.skipIf(not _util.py36, 'os.path support for Paths added in 3.6')
+    def test_path_obj_is_path(self):
+        # Arrange
+        from pathlib import Path
+        fp = Path('filename.ext')
```

Comment:
`fp` is often used for file pointers, shall we rename it to something else?

```suggestion
        test_path = Path('filename.ext')
```

Reply:
Sure. I only used `fp` to be consistent with the other path/strinType tests, I'm certainly not attached to the name.